packages/api/src/external/commonwell/admin/recreate-patients-at-hies.ts

```diff
@@ -102,9 +103,11 @@ export async function recreatePatientAtCW(
     const commonWell = makeCommonWellAPI(orgName, oid(orgOID));
     const queryMeta = organizationQueryMeta(orgName, { npi: facilityNPI });
 
+    const orgIdExcludeList = await getAllCQOrgsIds();
```

Comment:
The function where this is being called is executed once for each patient we're recreating. That alone is a good reason to pass the list as parameter instead of querying it for every patient.

We could move this all the way to the endpoint that calls it, so we don't have CW code loading CQ one.